CREDITS.md

```diff
@@ -18,7 +18,7 @@
 - Carlo Supina, [@cdsupina](https://github.com/cdsupina) – Dev Advocacy
 - Micah, [@tigleym](https://github.com/tigleym) – Dev Advocacy
 - Drake, [@Drakeyn](https://github.com/Drakeyn)– Media Content
-- Zyrafa, [@SirGiraffey](https://github.com/SirGiraffey) – Gameplay Design
+- Michał Grzegórzek - Kiaszewicz, [@SirGiraffey](https://github.com/SirGiraffey) – Gameplay Design
```

Comment:
It looks like the 'ł' character is also missing from the current font set we're using. #316 introduces a new font for the credits this should be fine once it's merged.